code/src/beetle.cpp

```diff
@@ -0,0 +1,75 @@
+#include <beetle.hpp>
+
+namespace r2d2::moving_platform {
+
+    beetle_c::beetle_c(r2d2::usart::usart_ports &usart_port, unsigned int baud_rate,
+             hwlib::pin_out &reset_pin, 
+             base_comm_c &comm):
+             moving_platform_c(comm),
+             qik_2s12v10_motorcontroller (usart_port, baud_rate, reset_pin)
+    {
+        
+        qik_2s12v10_motorcontroller.init();
+        qik_2s12v10_motorcontroller.get_configuration_parameter(1);
+
+    }
+    void beetle_c::set_speed(const int8_t &new_speed){
```

Comment:
there should be a space between the `)` and `{`

Reply:
Your commit has fixed this